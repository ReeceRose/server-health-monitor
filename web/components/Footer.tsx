import Link from 'next/link';

export default function Footer(): JSX.Element {
  return (
    <footer className="block py-4 absoulte">
      <div className="container px-24 mx-auto">
        <hr className="mb-4 border-b-1 border-blueGray-200" />
        <div className="flex flex-wrap items-center justify-center md:justify-between">
          <div className="w-full px-4 md:w-4/12">
            <div className="py-1 text-sm font-semibold text-center text-blueGray-500 md:text-left">
              Copyright Â© {new Date().getFullYear()}{' '}
              <Link href="https://github.com/PR-Developers/">
                <a
                  className="py-1 text-sm font-semibold text-blueGray-500 hover:text-blueGray-700"
                  target="_blank"
                  rel="noreferrer"
                >
                  PR-Developers
                </a>
              </Link>
            </div>
          </div>
          <div className="w-full px-4 md:w-8/12">
            <ul className="flex flex-wrap justify-center list-none md:justify-end">
              <li>
                <Link href="https://github.com/PR-Developers/server-health-monitor/">
                  <a
                    className="block px-3 py-1 text-sm font-semibold text-blueGray-600 hover:text-blueGray-800"
                    target="_blank"
                    rel="noreferrer"
                  >
                    GitHub
                  </a>
                </Link>
              </li>
            </ul>
          </div>
        </div>
      </div>
    </footer>
  );
}
