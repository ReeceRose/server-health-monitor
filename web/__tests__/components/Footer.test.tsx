import { render } from '@testing-library/react';

import Footer from '../../components/Footer';

test('Footer component matches snapshot', () => {
  const { asFragment } = render(<Footer />);
  expect(asFragment()).toMatchSnapshot();
});
