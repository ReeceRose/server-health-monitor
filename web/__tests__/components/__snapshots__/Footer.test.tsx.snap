// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component matches snapshot 1`] = `
<DocumentFragment>
  <footer
    class="block py-4 absoulte"
  >
    <div
      class="container px-24 mx-auto"
    >
      <hr
        class="mb-4 border-b-1 border-blueGray-200"
      />
      <div
        class="flex flex-wrap items-center justify-center md:justify-between"
      >
        <div
          class="w-full px-4 md:w-4/12"
        >
          <div
            class="py-1 text-sm font-semibold text-center text-blueGray-500 md:text-left"
          >
            Copyright Â© 2021 
            <a
              class="py-1 text-sm font-semibold text-blueGray-500 hover:text-blueGray-700"
              href="https://github.com/PR-Developers/"
              rel="noreferrer"
              target="_blank"
            >
              PR-Developers
            </a>
          </div>
        </div>
        <div
          class="w-full px-4 md:w-8/12"
        >
          <ul
            class="flex flex-wrap justify-center list-none md:justify-end"
          >
            <li>
              <a
                class="block px-3 py-1 text-sm font-semibold text-blueGray-600 hover:text-blueGray-800"
                href="https://github.com/PR-Developers/server-health-monitor/"
                rel="noreferrer"
                target="_blank"
              >
                GitHub
              </a>
            </li>
          </ul>
        </div>
      </div>
    </div>
  </footer>
</DocumentFragment>
`;
